.gitignore

```diff
@@ -1,4 +1,5 @@
 **/*.so
+.DS_Store
```

Comment:
Please also delete the .DS_store file from the repo

Reply:
what do you mean?